cmd/triggerRun/cmd/root.go

```diff
@@ -43,12 +54,51 @@ var (
 func init() {
 	rootCmd.Flags().StringVarP(&triggerFile, "triggerFile", "t", "", "Path to trigger yaml file")
 	rootCmd.Flags().StringVarP(&httpPath, "httpPath", "r", "", "Path to body event")
+	rootCmd.PersistentFlags().StringVarP(&kubeconfig, "kubeconfig", "k", "", "absolute path to the kubeconfig file")
 }
 
 func rootRun(cmd *cobra.Command, args []string) {
 	// TODO: Not implemented
 }
 
+func trigger(w io.Writer, triggerFile, httpPath string) error {
```

Comment:
curious to know where this function has been used :smile:

Reply:
Yeah, that's the intention. We could wire it up in this PR itself?